### DISCUSSION TITLE
Disabling personal Copilot subscription after getting license from organization.

### DISCUSSION THREAD

**Discussion by matthiasn-cdgnm** [community/community#49426] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I started a Copilot Subscription in my account which was still in the trial period. However, with the availability to apply individual licenses also for Team plan, my company assigned an organization-managed Copilot license to me. Now, I cannot access/cancel my personal license and I just got an email reminder, that my trial period is running out.

Do I still need to cancel my personal license (and how?) or is it automatically cancelled by being granted a license through my Organization?

Thanks!

**Comment by davecheney** [comment, marked as answer]
Your individual copilot subscription should have been cancelled when you were "adopted" by your organisation's Copilot plan. Please contact support who can assist you.

**Reply by matthiasn-cdgnm** [reply]
Thanks! I figured as much, but the email notification that my free trial is about to expire came after I was adopted by my company. I'll contact support to be safe :)

**Reply by matthiasn-cdgnm** [reply]
Support confirmed that the individual license is cancelled once you are adopted by an Org license. The emails regarding the trial period/individual license were sent in error.

**Comment by finnieb** [comment]
Dear Sir or Madam, I am hereby resigning my membership in your organization.  I suffer from Parkinson’s disease, and I don’t think that I will have the time or the energy to participate in the activities that I believe I will have perform if I remain a member.  

Sincerely,

James Yarnell
